src/app/space/analyze/analyze-overview/analyze-overview.component.ts

```diff
@@ -43,6 +44,13 @@ export class AnalyzeOverviewComponent implements OnInit, OnDestroy {
         this._myWorkItemsCard = true;
       }
     }));
+
+    this.userOwnsSpaceBool = this.userOwnsSpace();
```

Comment:
I'm wondering if it'd be cleaner to call the function here to set the `userOwnsSpace` boolean, and create a get method for retrieving the value (similar to how the `myWorkItemsCard` boolean is used). 

Something like:
`get userOwnsSpace: boolean () { return this._userOwnsSpace; }`